src/styles/blocks/main.scss

```diff
@@ -0,0 +1,27 @@
+.catalog {
+  --column-count: 1;
+
+  padding: 50px 40px;
+  display: grid;
+  grid-template-columns: repeat(var(--column-count), $product-card-width);
```

Comment:
Ensure that the variable `$product-card-width` is defined in your SCSS files. If it's missing, this could lead to compilation errors.